app/helpers/remote_execution_helper.rb

```diff
@@ -236,4 +243,25 @@ def load_template_from_task(template_invocation, target)
     return if task.nil?
     task.execution_plan.actions[1].try(:input).try(:[], 'script')
   end
+
+  def jobs_template_with_input(job_invocation)
+    template_name = Setting['remote_execution_job_invocation_template']
+    template = ReportTemplate.where(name: template_name).first
+    return if template.nil?
+    template_input = template.template_inputs.where(name: 'job_id').first
+    return if template_input.nil?
+    input_id = template_input.id
```

Comment:
I think this is unnecessary variable, `template_input.id` is quite readable, the same case as `job_invocation.id`